sdk/keyvault/azure-security-keyvault-common/src/keyvault_exception.cpp

```diff
@@ -12,7 +12,7 @@
 using namespace Azure::Security::KeyVault;
 using namespace Azure::Core::Http::_internal;
 
-Azure::Core::RequestFailedException _detail::KeyVaultException::CreateException(
+Azure::Core::RequestFailedException _internal::KeyVaultException::CreateException(
```

Comment:
`_detail` => `_internal` is intentional, right?

Reply:
yes